dist/topology-graph.css

```diff
@@ -21,22 +21,22 @@ kubernetes-topology-graph {
 
```

Comment:
afaik you shouldn't be changing files in the dist directory, they are generated from the root dir.